py3-cython.yaml

```diff
@@ -40,17 +40,38 @@ subpackages:
       provider-priority: ${{range.value}}
       provides:
         - ${{package.name}}
+      runtime:
         - cython
     pipeline:
       - uses: py/pip-build-install
         with:
           python: python${{range.key}}
       - uses: strip
 
+  - range: py-versions # This could just be 'cython' but we already have a package for that.
+    name: py${{range.key}}-${{vars.pypi-package}}-bin
+    description: Executable binaries for ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      provides:
+        - cython
+      provider-priority: ${{range.value}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr
+          mv ${{targets.contextdir}}/../py${{range.key}}-${{vars.pypi-package}}/usr/bin ${{targets.contextdir}}/usr
```

Comment:
Add a -bin subpackage inline test perhaps just `cython --version`

Reply:
I'm going to implement these changes over on https://github.com/wolfi-dev/os/pull/26844 because h5py still has a fair amount of deps to be fixed up and yarl can reasonably land soon. 

Alternatively I can split off cython and numpy and pickup pandas as well and do those in one PR which tackles some similar packages on our list.